dev-requirements.txt

```diff
@@ -10,8 +10,8 @@ kfp==1.8.9
 moto==2.2.12
 pyre-extensions==0.0.21
 pytorch-lightning==1.5.6
-ray==1.9.0
 s3fs==2021.10.1
+ray[default]
```

Comment:
can we pin the version here? we had a problem that it takes really huge time(40 minutes) to resolve dependencies for pip if versions are not pinned.

Reply:
Latest ray release is 1.9.2 https://pypi.org/project/ray/ 

Given ray 1.10 is coming out soon i would suggest updating this as soon as it come out, since it has log streaming and request header / cookie support for auth.

Reply:
Soundds good